documentation/guides/migrating-from-v8-to-v9.md

```diff
@@ -190,6 +190,10 @@ We replaced a few of the following filter function instances with color tokens i
 | `ms-high-contrast-color('button-text-background')`   | `buttonFace`            |
 | `ms-high-contrast-color('background')`               | `window`                |
 
+#### `px()`
+
+All `px` values are now automatically converted to `rem` through our postcss config with the `postcss-pxtorem` plugin.
```

Comment:
This is only true if the consumer has a postcss plugin that does this. This repo contains such a plugin but @shopify/postcss-plugin doesn't contain anything that does that, any app using sewing-kit would need to opt into using that plugin, which seems like a pretty disruptive migration suggestion.


Considering this is a migration guide for what to change in apps that consume polaris-react saying "the build in the  polaris-react repository" doesn't seem very relevant.

Reply:
🤦‍♀️ of course. I'll update this to suggest copying the existing function. Unless you have a better suggestion?

Reply:
Copying the existing function sounds like a good plan to me